spec/spec_helper.rb

```diff
@@ -12,6 +12,13 @@ module Gem
   end
 end
 
+if ENV.select {|k, _v| k =~ /TRAVIS/ }.any?
```

Comment:
Is this the same thing?

```suggestion
if ENV.keys.any? { |k| k.start_with?('TRAVIS_') }
```

Reply:
I think so, or we could even check for an exact value: https://docs.travis-ci.com/user/environment-variables/#default-environment-variables.

Reply:
By the way: lots of fanmail your way for all the great work and good cheer you bring to Bundler.